recipe/meta.yaml

```diff
@@ -0,0 +1,118 @@
+{% set name = "conda-store" %}
+{% set version = "2024.1.1" %}
+
+package:
+  name: {{ name|lower }}-split
+  version: {{ version }}
+
+source:
+  # This is only needed here in-repo; in conda-forge, sources from PyPI
+  - path: ../
+
+build:
+  number: 0
+
+outputs:
+  - name: conda-store
+    version: {{ version }}
+    build:
+      noarch: python
+      script: python -m pip install ./conda-store -vv --no-deps --no-build-isolation --no-index
+      entry_points:
+        - conda-store = conda_store.__main__:main
+    requirements:
+      host:
+        - python >=3.8
+        - pip
+        - hatchling >=1.14.0
+        - hatch-vcs
+      run:
+        - __linux  # [linux]
+        - __osx    # [osx]
+        - __win    # [win]
+        - aiohttp
+        - click
+        - python >=3.8
+        - rich
+        - ruamel.yaml
+        - yarl
+      run_constrained:
+        - {{ pin_subpackage('conda-store-server', min_pin='x.x.x', max_pin='x.x.x') }}
+
+    test:
+      imports:
+        - conda_store
+      commands:
+        - conda-store --help
+
+  - name: conda-store-server
+    version: {{ version }}
+    build:
+      noarch: python
+      script: python -m pip install ./conda-store-server -vv --no-deps --no-build-isolation --no-index
+      entry_points:
+        - conda-store-server = conda_store_server.server.__main__:main
+        - conda-store-worker = conda_store_server.worker.__main__:main
```

Comment:
Entry points were missing, which could cause issues with the executables in non-Linux platforms.